src/index.html

```diff
@@ -8,6 +8,56 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Stars</h1>
+    <div class="star_container stars--0">
+      <div class="stars_star"></div>
```

Comment:
```suggestion
    <div class="stars stars--0">
      <div class="stars__star"></div>
```
Add a default class of stars without modifier too.
And provide two underlines, so it's clear that it is an element of stars block